setup.sh

```diff
@@ -3,6 +3,14 @@
 # Setup network configuration for a CentOS 6.6 virtual machine on VirtualBox
 # Please see directions at https://github.com/enterprisemediawiki/meza1
 
+# Get host-only IP address
+while [ -z "$ipaddr" ]
```

Comment:
By using a while loop, this doesn't allow the user to just hit "enter" and use the default value in line 10.

![image](https://cloud.githubusercontent.com/assets/6728676/9413337/e022e830-47f6-11e5-871d-ac984c22d47b.png)

What do you think about doing the following instead? Replace lines 7-11 with the following:

```
default_ipaddr="192.168.56.56"
echo -e  "Enter desired IP address or just hit enter to use the default value ([$default_ipaddr])"
read ipaddr
ipaddr=${ipaddr:-$default_ipaddr}
echo "You chose" $ipaddr
```

![image](https://cloud.githubusercontent.com/assets/6728676/9413800/da3299e0-47f9-11e5-92f5-40645db9dd77.png)

See [this SO post](http://stackoverflow.com/questions/2642585/read-a-variable-in-bash-with-a-default-value) and [this tutorial](http://www.network-theory.co.uk/docs/bashref/ShellParameterExpansion.html) for background on syntax.